src/main/webapp/app/quiz/participate/quiz-scoring-infostudent-modal/quiz-scoring-info-student-modal.component.ts

```diff
@@ -0,0 +1,182 @@
+            this.inTotalSelectedWrongOptions = this.multipleChoiceWrongAnswerChosen + this.forgottenMultipleChoiceRightAnswers;
+            this.differenceMultipleChoice = this.inTotalSelectedRightOptions - this.inTotalSelectedWrongOptions;
+
+            if(this.inTotalSelectedRightOptions == 1){
+                this.rightOption = this.translateService.instant(translationBasePath + 'option');
+            } else {
+                this.rightOption = this.translateService.instant(translationBasePath + 'options');
+            }
+
+            if(this.inTotalSelectedWrongOptions == 1){
+                this.wrongOption = this.translateService.instant(translationBasePath + 'option');
+            } else {
+                this.wrongOption = this.translateService.instant(translationBasePath + 'options');
+            }
+    }
+
+    /**
+     * counts the variables for Drag and Drop Questions
+     */
+    private countDragandDrop() {
+        const translationBasePath = 'arTeMiSApp.quizExercise.explanationText.';
+        const dndQuestion = this.question as DragAndDropQuestion;
+            this.amountOfDragAndDropZones = dndQuestion.dropLocations.length;
+            this.wronglyMappedDragAndDropItems = this.amountOfDragAndDropZones - this.correctlyMappedDragAndDropItems;
+            this.differenceDragAndDrop = this.correctlyMappedDragAndDropItems - this.wronglyMappedDragAndDropItems;
+
+        if(this.correctlyMappedDragAndDropItems == 1){
+            this.rightMap = this.translateService.instant(translationBasePath + 'item');
+        } else {
+            this.rightMap = this.translateService.instant(translationBasePath + 'items');
+        }
+
+        if(this.wronglyMappedDragAndDropItems == 1){
+            this.wrongMap = this.translateService.instant(translationBasePath + 'item');
+        } else {
+            this.wrongMap = this.translateService.instant(translationBasePath + 'items');
+        }
+    }
+
+    /**
+     * counts the variables for Short Answer Questions
+     */
+    private countShortAnswer() {
+        const translationBasePath = 'arTeMiSApp.quizExercise.explanationText.';
+        const shortAnswer = this.question as ShortAnswerQuestion;
+            this.shortAnswerSpotCount = shortAnswer.spots.length;
+            this.shortAnswerCorrectAnswers = this.ShortAnswerText.filter(option => option.isCorrect).length;
+            this.shortAnswerWrongAnswers = this.shortAnswerSpotCount - this.shortAnswerCorrectAnswers;
+            this.differenceShortAnswer = this.shortAnswerCorrectAnswers - this.shortAnswerWrongAnswers;
+
+        if(this.shortAnswerCorrectAnswers == 1){
+            this.rightGap = this.translateService.instant(translationBasePath + 'textgap');
+        } else {
+            this.rightGap = this.translateService.instant(translationBasePath + 'textgaps');
+        }
+
+        if(this.shortAnswerWrongAnswers == 1){
```

Comment:
the spacing is not correct. Please be consistent. You might want to use the linter

Reply:
This is the case in several places in this file!